frontend/simple/views/Invite.vue

```diff
@@ -131,16 +130,12 @@ export default {
                   { contractID: groupId, type: inviteToGroup.type(), action: JSON.stringify(inviteToGroup.data()) }
                 ],
                 initiator: this.loggedIn.name,
-                initiationDate: new Date().toISOString()
+                initiationDate: new Date().toString()
               },
               groupId
             )
             await sbp('backend/publishLogEntry', proposal)
           } else {
-            // TODO: place these in SBP calls, consider using ArchivedTransactionQueue
-            // NOTE: avoiding use of ArchivedTransactionQueue is better than using it. Here for example,
-            //       we could make it so that a user can be invited to a group multiple times, and the system
-            //       just accepts/cares about the most recent invite only and forgets the previous one
```

Comment:
Why was this comment deleted?

Reply:
because they were placed in SBP calls. I can add back the transactionqueue part, but I assumed that was captured in its own isse

Reply:
yes please, the NOTE should be restored